[Speed] Update http2-to-origin.md (#13522)
cloudflare/cloudflare-docs: content/speed/optimization/protocol/http2-to-origin.md

## Enable HTTP/2 to Origin

At Cloudflare, HTTP/2 connection is enabled by default for Free, Pro and Business customers. It will also be gradually enabled for Enterprise customers from March 19, 2024. For more details, refer to the warning below. 

Follow these steps to enable HTTP/2 to Origin:

1.  Log into the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
2.  Choose the domain that will use HTTP/2 to Origin.
3.  Select **Speed > Optimization**.
4.  Navigate to the **Protocol Optimization** tab and under **HTTP/2 to Origin** set the toggle to **On**.

If the toggle is disabled, or the origin does not support HTTP/2, Cloudflare will initiate an HTTP/1.1 connection.

> {{<Aside type="warning" header="Important">>}}

Reviewer: ```suggestion
{{<Aside type="warning" header="Important">}}
```

## Enable HTTP/2 to Origin

At Cloudflare, HTTP/2 connection is enabled by default for Free, Pro and Business customers. It will also be gradually enabled for Enterprise customers from March 19, 2024. For more details, refer to the warning below. 

Follow these steps to enable HTTP/2 to Origin:

1.  Log into the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
2.  Choose the domain that will use HTTP/2 to Origin.
3.  Select **Speed > Optimization**.
4.  Navigate to the **Protocol Optimization** tab and under **HTTP/2 to Origin** set the toggle to **On**.

If the toggle is disabled, or the origin does not support HTTP/2, Cloudflare will initiate an HTTP/1.1 connection.

{{<Aside type="warning" header="Important">>}}
Starting March 19, 2024, we plan to gradually enable HTTP/2 to origin by default for all Enterprise customers. 

> We will connect to servers who announce support of HTTP/2 connections via [ALPN](https://blog.cloudflare.com/introducing-http2). 

Reviewer: ```suggestion
We will connect to servers who announce support of HTTP/2 connections via [ALPN](https://blog.cloudflare.com/introducing-http2). If you are unsure if your server supports HTTP/2, we suggest checking your origin server’s documentation or using a testing tool for HTTP/2 implementation (for example, [h2spec](https://github.com/summerwind/h2spec)). For servers that do not announce HTTP/2 support, nothing will change from how Cloudflare connects to your origin server today. 
```

> If you are unsure if your server supports HTTP/2, we suggest checking your origin server’s documentation or using a testing tool for HTTP/2 implementation (for example, [h2spec](https://github.com/summerwind/h2spec)). For servers that do not announce HTTP/2 support, nothing will change from how Cloudflare connects to your origin server today. 

Reviewer: ```suggestion
```

## Enable HTTP/2 to Origin

> At Cloudflare, HTTP/2 connection is enabled by default for Free, Pro and Business customers. 

Reviewer: ```suggestion
At Cloudflare, HTTP/2 connection is enabled by default for Free, Pro and Business customers. It will also be gradually enabled for Enterprise customers from March 19, 2024. For more details, refer to the warning below. 

Follow these steps to enable HTTP/2 to Origin:
```

> It will also be gradually enabled for Enterprise customers from March 19, 2024, see below. 

Reviewer: ```suggestion
```

> Please follow these steps to enable it:

Reviewer: ```suggestion
```

If the toggle is disabled, or the origin does not support HTTP/2, Cloudflare will initiate an HTTP/1.1 connection.

> {{<Aside type="warning">}}

Reviewer: ```suggestion
{{<Aside type="warning" header="Important">>}}
```

Starting March 19, 2024, we plan to gradually enable HTTP/2 to origin by default for all Enterprise customers. 

We will connect to servers who announce support of HTTP/2 connections via [ALPN](https://blog.cloudflare.com/introducing-http2). 

> If you are unsure if your server supports HTTP/2, we suggest checking your origin server’s documentation or using any number of tools (example [here](https://github.com/summerwind/h2spec)). For servers that do not announce HTTP/2 support, nothing will change from how Cloudflare connects to your origin server today. 

Reviewer: ```suggestion
If you are unsure if your server supports HTTP/2, we suggest checking your origin server’s documentation or using a testing tool for HTTP/2 implementation (for example, [h2spec](https://github.com/summerwind/h2spec)). For servers that do not announce HTTP/2 support, nothing will change from how Cloudflare connects to your origin server today. 
```

> If you do not want Cloudflare to connect to your origin via HTTP/2 you can either disable ALPN HTTP/2 support on your origin server, or set [origin_max_http_version to “1” via the API](/api/operations/zone-cache-settings-change-origin-max-http-version-setting) by March 19, 2024.

Reviewer: ```suggestion
If you do not want Cloudflare to connect to your origin via HTTP/2 you can either disable ALPN HTTP/2 support on your origin server, or set `origin_max_http_version` to `1` via the [API](/api/operations/zone-cache-settings-change-origin-max-http-version-setting) by March 19, 2024.
```

> Once the HTTP/2 to origin is enabled, you can navigate to **Speed > Optimization > Protocol Optimization** in the Cloudflare dashboard and set the toggle to Off to disable it. 

Reviewer: ```suggestion
Once the HTTP/2 to origin is enabled, you can navigate to **Speed > Optimization > Protocol Optimization** in the Cloudflare dashboard and set the toggle to **Off** to disable it. 
```
